packages/framework/aqueduct/src/container-services/containerServices.ts

```diff
@@ -13,14 +13,21 @@ import {
 } from "@fluidframework/runtime-utils";
 
 // TODO: should this just be "s"?
+/**
+ * @deprecated See {@link @fluidframework/synthesize#DependencyContainer}
+ */
 export const serviceRoutePathRoot = "_services";
 
+/**
+ * @deprecated See {@link @fluidframework/synthesize#DependencyContainer}
+ */
 export type ContainerServiceRegistryEntries = Iterable<
 	[string, (runtime: IContainerRuntime) => Promise<FluidObject>]
 >;
 
 /**
  * This class is a simple starter class for building a Container Service. It simply provides routing
+ * @deprecated See {@link @fluidframework/synthesize#DependencyContainer}
  */
 export abstract class BaseContainerService implements IFluidRouter {
 	public get IFluidRouter() {
```

Comment:
I also like to put the deprecated tag on all public members of the class (including constructor) for extra visibility.